src/java/picard/util/BaitDesigner.java

```diff
@@ -39,11 +39,36 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Designs baits or oligos for hybrid selection reactions.",
-        usageShort = "Designs baits or oligos for hybrid selection reactions.",
-        programGroup = None.class
-)
+                usage = BaitDesigner.USAGE_SUMMARY + BaitDesigner.USAGE_DETAILS,
+                usageShort = BaitDesigner.USAGE_SUMMARY,
+                programGroup = None.class
+        )
 public class BaitDesigner extends CommandLineProgram {
+static final String USAGE_SUMMARY = "Designs baits or oligonucleotides for hybrid selection reactions.  ";
+static final String USAGE_DETAILS = "<p>This tool uses an interval file corresponding to discrete sequences to design synthetic \"baits\" for hybrid selection experiments.  " +
+        "" +
+        "Hybrid selection enriches a genomic sample for specific or \"target\" sequences e.g. exons.  " +
+        "For additional information on hybrid selection, please refer to the GATK " +
+        "<a href=\"http://gatkforums.broadinstitute.org/gatk/discussion/6331\">Dictionary</a> entry.</p>" +
+        "" +
+        "Briefly, users input a (TARGET) interval list file indicating the sequences of interest e.g. exons with their respective coordinates, a reference sequence, and a unique identifier string (DESIGN_NAME).  " +
+        "The program outputs interval list files of both \"bait\" and \"target\" sequences as well as the actual bait sequences in fasta format." +
+        "  At least two baits are output for each target sequence with greater numbers for larger intervals.  Although the default values for both bait size" +
+        " (120 bases) and offsets (80 bases) are suitable for most applications, users have the option to customize these values. Offsets represent the distance between sequential baits on a contiguous stretch of target DNA sequence.  "+
+        "" +
+        "This program also outputs files consisting of a pooled set of 55,000 (default) oligonucleotides representing all of the baits " +
+        "redundantly.  This redundancy achieves a uniform concentration of olignucleotides for synthesis by a vendor as well as equal numbers" +
```

Comment:
This is the question that I had for Tim that he never answered.  Yossi thought that each oligo is replicated multiple times such that each batch of oligos achieves a uniform concentration of 55K oligos/tube.  If there are only 100 unique oligos designed by the program for a given experiment, the file will contain 550 copies of each of the 100 unique oligos to achieve 55K oligos total.  At least I think that is what he meant.